cmd/main.go

```diff
@@ -56,6 +56,8 @@ func main() {
 	syscall.Umask(0)
 
 	kubeconfig := flag.String("kubeconfig", "", "Path to Kubeconfig file with authorization and master location information.")
+	volumeRoundingEnabled := flag.Bool("rounding-enabled", true, "When enabled volumes will be rounded up if less than 'minVolumeSizeMB'")
+	minVolumeSizeMB := flag.Int("min-size", 51200, "The minimum size for a block volume. By default OCI only supports block volumes > 50GB")
```

Comment:
`s/min-size/min-volume-size/`?

Reply:
Also using a string and `k8s.io/apimachinery/pkg/api/resource` rather than an int would be preferable.